cves/kernel/CVE-2018-7480.yml

```diff
@@ -149,10 +153,10 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then please
     explain where you looked.
-  answer:
-  automated:
-  contest:
-  developer:
+  answer: 2018-02-26
```

Comment:
Would be great if you can also explain how this vulnerability was found not just the date. Then, update whether it was by a developer or automated